### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by yashk0thari**
#85 I've added missing speaker's profiles

### CODE REVIEW

**Comment by yashk0thari** [comment]
if there are no conflicts, can we please merge

**Review by anshumanv — COMMENTED** [review]
Please specify what you did in the PR, give test link.

**Comment by yashk0thari** [comment]
https://beta-king.github.io/Beta-King2017.fossasia.org/ --> preview link

**Comment by yashk0thari** [comment]
Done!!

File: index.html [hunk 1]
```diff
@@ -1290,6 +1296,7 @@ <h1>Featured Speakers</h1>
 											<a target="default" href="http://dev.mysql.com/"><i class="icon fa fa-external-link"></i></a>
                       <a target="default" href="https://github.com/JesperWisborgKrogh"><i class="icon fa fa-github"></i></a>
 											<a target="default" href="https://www.linkedin.com/in/jesperwisborgkrogh"><i class="icon social_linkedin"></i></a>
+											<a target="default" href="https://www.facebook.com/sudheesh001"><i class="icon social_facebook"></i></a>
```
Comment (anshumanv) [inline index.html]: Wrong link

**Comment by yashk0thari** [comment]
Made necessary changes, they should be correct now :D sorry for the inconvenience

**Comment by yashk0thari** [comment]
if everything is alright, can we merge ? :D

**Review by anshumanv — APPROVED** [review]

**Comment by yashk0thari** [comment]
Are anymore checks left ??

**Comment by yashk0thari** [comment]
FYI, i've fixed the font error in 2016's task and also review 2015's task, thank you so much :D

**Comment by anshumanv** [comment]
>Are anymore checks left ??

Please wait for other mentors to review.

**Comment by yashk0thari** [comment]
ohkk :D

**Comment by yashk0thari** [comment]
how much longer ??

**Comment by yashk0thari** [comment]
thank you :D

**Pull request merged**